### ISSUE TITLE
How to enable version switcher?

### ISSUE THREAD

**Issue by rqiu25** [python-restx/flask-restx#129]
Hi team, I go through the doc but I didn't find a way to enable the version switcher in swagger, what I want to achieve is to present different swagger.json based on the version.
I've already used namespace for the API:
foo/v1/bar
foo/v1.1/bar
but the json contains the whole API v1 and v1.1, and it's kind of messy.
Thanks in advance

**Comment by alexandrevillard** [comment]
Hey, have you found a solution for this?

**Comment by rqiu25** [comment]
Yes, I was able to provide two different swagger.json under two different versions by using blueprints: you define the different api version in two different blueprints than register them in the root Flask object instance.
